ImportBlocks/ImportBlocks.cs

```diff
@@ -42,54 +33,65 @@ private bool OnExport(string[] args)
                         fs.Seek(sizeof(uint), SeekOrigin.Begin);
                     }
                     else
+                    {
+                        byte[] buffer = new byte[sizeof(uint)];
```

Comment:
Maybe declare `buffer` outside this `if else`

Reply:
will do

Reply:
will do